### ISSUE TITLE
finetuning pretrain model, test accuracy doesn't increase to 0.9+ 

### ISSUE THREAD

**Issue by guoqiang01486** [BVLC/caffe#6162]
l am a newer to caffe, this is a wired things when i train my model, for a same solver.prototxt, if l train from no pretrain model, its test accuracy can increase to 0.9+, however if l finetune from pretrain model, its accuracy fluctuate to 0.5-0.6, how can l solve the problem, making finetune model test accuracy increase to 0.9, please help me!!! thank you very much.

following is my command and configure file:

1 train:

./build/tools/caffe train --solver=solver_cmp.prototxt -gpu 0

2 finetune:

./build/tools/caffe train --solver=solver_cmp.prototxt --weights=pruned_sqznet.caffemodel -gpu 0

3 solver_cmp.prototxt

net: "pruned_sqznet.prototxt" #we typically do `cd SqueezeNet_v1.0; caffe train <args>`
test_iter: 80 #not subject to iter_size
test_interval: 100
base_lr: 0.01
type: "Nesterov"
display: 40
max_iter: 100000
#iter_size: 16 #global batch size = batch_size * iter_size
#lr_policy: "fixed"
#gamma: 0.0001
#power: 0.75
lr_policy: "poly"
power: 1.0 #linearly decrease LR
momentum: 0.9
weight_decay: 0.005
snapshot: 10000
snapshot_prefix: "examples/crowd/models/cmp_fine"
random_seed: 42
solver_mode: GPU
average_loss: 40

**Comment by Noiredd** [comment]
Maybe you should decrease your learning rate. Please post questions like this (usage, installation, or modeling, or other requests for help) to the [caffe-users list][1] instead of issues. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe. Please read the [guidelines for contributing][2] before submitting an issue or a pull request.

 [1]: https://groups.google.com/forum/#!forum/caffe-users
 [2]: https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md
